src/components/Header/Header.tsx

```diff
@@ -53,6 +56,7 @@ export const Header = () => {
         </MastheadBrand>
       </MastheadMain>
       <MastheadContent>
+        {user?.identity?.account_number && !isFedRamp() && isDebuggerEnabled && ReactDOM.createPortal(<Debugger user={user} />, document.body)}
```

Comment:
Just a nitpick:  I don't think we want the debugger to be part of a Header component. I would use the `RootApp` or its direct child as a host of the debugger component.

And because this component will be rarely used by actual users, I would lazily load it and wrap it in Suspens to make sure the code is only loaded when requested.